QueryBuilder/Compilers/SqliteCompiler.cs

```diff
@@ -66,5 +67,42 @@ protected override string CompileBasicDateCondition(SqlResult ctx, BasicDateCond
 
             return sql;
         }
+
+        private class AggregateAnyValueColumn: SqlKata.AggregateAnyValueColumn
+        {
+            public override string Compile(SqlResult ctx)
+            {
+                return $"COALESCE(NULL, {new Column { Name = Column }.Compile(ctx)}) {ctx.Compiler.ColumnAsKeyword}{ctx.Compiler.WrapValue(Alias ?? Type)}";
+            }
+        }
+
+        protected override string CompileColumns(SqlResult ctx)
+        {
+            /**
+             * This replaces all "any_value" aggregates by the COALESCE function
+             * but this requires a group by clause (anything will do).
+             */
+            if (!ctx.Query.HasComponent("group") && ctx.Query.Clauses.Any(clause => clause is SqlKata.AggregateAnyValueColumn aggregate))
+            {
+                ctx.Query.GroupByRaw("\"\"");
+            }
+            ctx.Query.Clauses = ctx.Query.Clauses.Select(clause =>
+            {
+                if (clause is SqlKata.AggregateAnyValueColumn column)
+                {
+                    return new AggregateAnyValueColumn
+                    {
+                        Engine = column.Engine,
+                        Component = column.Component,
+                        Column = column.Column,
+                        Alias = column.Alias,
+                        Distinct = column.Distinct,
+                    };
```

Comment:
Can we to this in an `AggregateAnyValueColumn` constructor? Not a big deal now but if we ever build a generic replace helper it will fit better.

Reply:
b2c9694acf9d496d019419752bfd70a7489dcfda